crates/sui-framework/packages/bridge/sources/committee.move

```diff
@@ -98,18 +127,122 @@ module bridge::committee {
             // get committee signature weight and check pubkey is part of the committee
             let member = vec_map::get(&self.members, &pubkey);
             if (!member.blocklisted) {
-                threshold = threshold + member.voting_power;
+                threshold = threshold + member.stake_amount;
             };
             i = i + 1;
             vec_set::insert(&mut seen_pub_key, pubkey);
         };
-        assert!(threshold >= required_threshold, ESignatureBelowThreshold);
+        assert!(threshold >= required_stake, ESignatureBelowThreshold);
     }
 
+    public(friend) fun register(
```

Comment:
do we need proof of possession?

Reply:
no it's a bls thing

Reply:
So we will just accept any pub key provided by the validator without any verification? in some rare case if validator provides a wrong pub key by mistake, it will be registered and they won’t be able to sign any messages. Is that ok?

Reply:
i'm afraid that's the case (except that at least we can assert pub key bytes length) until v2.
In practice validators will register using provided cli, and we will do thorough checks there